### PR TITLE
fulcrum: init at 1.6.0

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

- new package - bitcoin blockchain indexer

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [x] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/blockchains/fulcrum/default.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+  dontWrapQtApps = true; # no GUI
+
+  buildInputs = [ qtbase rocksdb zeromq ];
+
+  meta = with lib; {
+    description = "Fast & nimble SPV server for Bitcoin Cash & Bitcoin BTC";
+    homepage = "https://github.com/cculianu/Fulcrum";
+    maintainers = with maintainers; [ prusnak ];
+    license = licenses.mit;
```
Comment (mmilata) [inline pkgs/applications/blockchains/fulcrum/default.nix:32]: Seems to be GPLv3: https://github.com/cculianu/Fulcrum/blob/master/LICENSE.txt

**Review by mmilata — COMMENTED** [review]
Prints a bunch of `/nix/store/2kh3c4v2vf6d6xg6c9n8zvfpwf3zzwca-bash-5.1-p12/bin/bash: line 1: git: command not found` during build but finishes successfully. Seems to use the system librocksdb despite the documentation suggesting that you need to explicitly tell it not to use the bundled static library. `Fulcrum --help` runs fine, however the admin script doesn't:
```
$ ./result/bin/FulcrumAdmin
/usr/bin/env: ‘python3’: No such file or directory
```

**Comment by prusnak** [comment]
@mmilata addressed both things in d77eae8f75ca8fa48c90401ab2a60c3a0a4bad45

**Review by mmilata — APPROVED** [review]
LGTM!

**Comment by prusnak** [comment]
@GrahamcOfBorg build fulcrum

**Pull request merged**